ddl/reorg.go

```diff
@@ -102,12 +105,17 @@ func (c *mockContext) ClearValue(key fmt.Stringer) {
 	delete(c.m, key)
 }
 
+func (c *mockContext) GetSessionVars() *variable.SessionVars {
+	return c.sessionVars
+}
+
 func (d *ddl) newMockContext() context.Context {
 	c := &mockContext{
-		store: d.store,
-		m:     make(map[fmt.Stringer]interface{}),
+		store:       d.store,
+		m:           make(map[fmt.Stringer]interface{}),
+		sessionVars: variable.NewSessionVars(),
 	}
-
+	c.sessionVars.SetStatusFlag(mysql.ServerStatusAutocommit, false)
```

Comment:
It was false before, but `NewSessionVars` set autocommit to true.
Set it to false so the test works the same as before.